stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/GsonChatApi.kt

```diff
@@ -116,6 +111,20 @@ internal class GsonChatApi(
         }
     }
 
+    override fun sendImage(
+        channelType: String,
+        channelId: String,
+        file: File,
+        callback: ProgressCallback,
+    ): Unit = fileUploader.sendImage(
```

Comment:
Perhaps we should make this (the `sendImage` method we're calling into) blocking as well? It's the only remaining async method in `FileUploader`. Some others return `Unit` too, but they are still synchronous.